extensions/amp-story-interactive/0.1/amp-story-interactive-img.css

```diff
@@ -58,11 +59,35 @@
 .i-amphtml-story-interactive-img-option-img {
   border-radius: 50% !important;
   overflow: hidden !important;
-  position: absolute !important;
-  top: 0 !important;
-  left: 0 !important;
   width: 100% !important;
   height: 100% !important;
   background-position: center !important;
   background-size: cover !important;
 }
+
+.i-amphtml-story-interactive-option-selected
+  .i-amphtml-story-interactive-img-option-img {
+  box-shadow: inset 0 0 0 0.25em var(--i-amphtml-interactive-option-post-color) !important;
+}
+
+.i-amphtml-story-interactive-post-selection.i-amphtml-story-interactive-has-data
+  .i-amphtml-story-interactive-img-option-img::before {
+  background-color: var(--i-amphtml-interactive-option-post-color) !important;
+  content: '' !important;
+  display: block !important;
+  width: 100% !important;
+  height: 100% !important;
+  margin-top: calc(100% - var(--option-percentage)) !important;
```

Comment:
Optional:
This is okay for now but when animating it, `transform: scaleY()` will need to be used for performance.